src/main/java/seedu/address/model/listing/Listing.java

```diff
@@ -160,6 +160,14 @@ public boolean isSameListing(Listing otherListing) {
                 || otherListing.id.equals(this.id));
     }
 
+    /**
```

Comment:
In line 29 isit possible to change  private final List<Person> interestedClients, to List<Client> interestedClients to be more specific?